src/applications/gi/utils/helpers.js

```diff
@@ -113,7 +126,7 @@ export const handleScrollOnInputFocus = fieldId => {
 };
 
 export const formatCurrency = value => {
-  if (isNaN(value)) {
+  if (Number.isNaN(value)) {
```

Comment:
Do we need this change? This renders "$NaN" in the table instead of numbers.

Reply:
Solved. Platform had a warning. It was wrong.